tasks/main.yml

```diff
@@ -31,25 +31,29 @@
     name: "{{ item }}"
     state: present
   with_items: "{{ ice_pip_dependencies }}"
-  when: "ice_pip_dependencies and (ice_python_wheel | length < 1)"
+  when: >-
```

Comment:
The duplicate blocks and the functionality makes me wonder whether it is time to split `tasks/main.yml` into `tasks/install-runtime.yml` and `tasks/install-python.yml` with a top-level include.

Reply:
I thought since this role is fairly small it's not worth splitting into two files (or three if you want to follow the same pattern for `ice_devel_packages`).